README.md

```diff
@@ -9,13 +9,23 @@ The minimal recommended setup for an application using [Cesium](https://cesiumjs
 	npm install
 	npm start
 
+Navigate to `localhost:8080`.
+
 ##### Available Scripts
 
 * `npm start` - Runs a webpack build with `webpack.config.js` and starts a development server
 * `npm run build` - Runs a webpack build with `webpack.config.js`
 * `npm run release` - Runs an optimized webpack build with `release.webpack.congif.js`
 
+##### Configurations
+
+We've included two webpack configuration files in this repository. `webpack.config.js` contains the minimal recommended configuration for getting setup, and configuration for running the development server. `release.webpack.config.js` contains  an optimized configuration for production use.
```

Comment:
@ggetz I would prefer `webpack.release.config.js` or `webpack.prod.config.js` for easier discoverability from the user and from tooling :smirk: